src/ray/raylet/scheduling/cluster_task_manager.h

```diff
@@ -84,6 +84,17 @@ class ClusterTaskManager : public ClusterTaskManagerInterface {
                       rpc::RequestWorkerLeaseReply::SCHEDULING_CANCELLED_INTENDED,
                   const std::string &scheduling_failure_message = "") override;
 
+  /// Attempt to cancel already queued tasks submitted by the owner.
```

Comment:
We shouldn't apply this if the task is a detached actor creation task (it doesn't fate share with the owner). Can you also add a test case for this one?